src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -439,6 +439,35 @@ class IndexLogEntryTest extends SparkFunSuite with SQLHelper with BeforeAndAfter
     FileUtils.deleteDirectory(tempDir.toFile)
   }
 
+  test("Directory.merge Test: test if merge works as expected.") {
+    // File Structure
+    // testDir/a/f1
```

Comment:
btw, you also have `temp`.

Reply:
thanks @imback82 for the catch. I have fixed this test as well as other tests to reflect the correct data structure. I will put comments at those places to make it clear.